Podfile.lock

```diff
@@ -36,6 +37,7 @@ PODS:
     - DTFoundation/Core
   - DTFoundation/UIKit (1.7.18):
     - DTFoundation/Core
+  - ffmpeg-kit-ios-audio (4.4)
```

Comment:
This (and below) need updating for the switch to 4.4.LTS. I get
> [!] The platform of the target `Riot` (iOS 11.0) may not be compatible with `ffmpeg-kit-ios-audio (4.4)` which has a minimum requirement of iOS 12.1.
Until I've done that.

Reply:
Hmm.. I don't think we commit changes to the lock file. I think you can safely 'pod update'  and override it on your side and you'll also need to run an xcodegen to get that new deployment target in..
Let me know if you still have problems.

Reply:
Sorry, yes I meant, updating via `pod` rather than directly. (Only that as it is right now `pod install` gave me that error - that said if you're making a custom Podfile for this, presumably the lock file work itself out anyway.